packages/prices/utils/price.ts

```diff
@@ -177,7 +193,9 @@ export const formatPrice = (
 		beforeDecimal,
 		thousandSeparator
 	) }${
-		afterDecimal ? `${ decimalSeparator }${ afterDecimal }` : ''
+		afterDecimal || minorUnit > 0
```

Comment:
Since we have a value for `afterDecimal`, and we are passing `minorUnit`, how about we remove the conditional here and just format it in the other util you wrote? We don't need to check twice.

Reply:
Great suggestion! I've updated the code accordingly.